src/Tools/CreateTools/Change/CreateMultiLineChange.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  Color,
+  GeomItem,
+  LinesMaterial,
+  FatLinesMaterial,
+  Lines,
+  Vec3,
+  Vec3Attribute,
+  TreeItem,
+  Xfo,
+} from '@zeainc/zea-engine'
+import UndoRedoManager from '../../../UndoRedo/UndoRedoManager'
+import CreateGeomChange from './CreateGeomChange'
+
+/**
+ * Class representing a create line change.
+ *
+ * **Events**
+ * * **updated:** Triggered when the change is updated
+ *
+ * @extends CreateGeomChange
+ */
+class CreateMultiLineChange extends CreateGeomChange {
+  line: Lines
+  stage: number
+  vertices: Vec3[] = []
+
+  /**
+   * Create a create line change.
+   *
+   * @param parentItem - The parentItem value.
+   * @param xfo - The xfo value.
+   * @param color - The color value.
+   * @param thickness - The thickness value.
+   */
+  constructor(parentItem: TreeItem, xfo: Xfo, color: Color, thickness = 0.001) {
```

Comment:
lets not pass in color and thickness for now. We can add it back later once we have that working better.